extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -275,7 +275,7 @@ export class ResourceTypePickerDialog extends DialogBase {
 			if (messages.length > 1) {
 				messages = messages.map(message => `•	${message}`);
 			}
-			messages.push(localize('deploymentDialog.VersionInformationDebugHint', "You will need to restart Azure Data Studio if the tools are installed manually after Azure Data Studio is launched to pick up the updated PATH environment variable. You may find additional details in 'Deployments' output channel"));
+			messages.push(localize('deploymentDialog.VersionInformationDebugHint', "You will need to restart Azure Data Studio if the tools are installed manually or eula is accepted manually to pick up the change. You may find additional details in 'Deployments and azdata' output channels"));
```

Comment:
`details in the 'Deployments' and 'azdata' output channels` - to make it clear that these are separate channels